budgetportal/settings.py

```diff
@@ -203,7 +205,7 @@
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
         'DIRS': [
-            str(ROOT_DIR.path("_includes"))
+            str(ROOT_DIR.path("budgetportal/templates")),
```

Comment:
is this even needed now that it's where it's expected?
https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#include